### PR TITLE
v4 Blogpost

### PR DESCRIPTION
**Pull request by altafan**
This is a first version of the blogpost for the upcoming Ark v4.

Please @tiero  review.

### CODE REVIEW

**Comment by altafan** [comment]
can't see why the preview build is failing

File: blog/2024-11-20-release-v0.4.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+authors: [altafan]
+description: Release ARK v0.4, featuring new address encoding, Ark notes, no required settlement, and live transaction feed.
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:3]: Redo

File: blog/2024-11-20-release-v0.4.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- Users can restore their off-chain balances when they come back online in the future
+
+This feature enhances the usability of Ark for users with intermittent connectivity, ensuring they don't miss out on the benefits of the network.
+
+### 🔄 No Required Settlement
+
+We've streamlined the transaction process:
+
+- VTXOs received out of round no longer need to be settled to be spent in another out-of-round transaction
+- This improvement reduces friction and increases the speed of off-chain transactions
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:45]: This actually enabled you to create a "chain" of "pending" payments, enabling a form of fast and trust minimized payment protocol that don't require the Server to allocate liquidity for every request.

File: blog/2024-11-20-release-v0.4.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+### 📡 Live Transaction Feed
+
+Real-time updates are now a reality with our new live transaction feed:
+
+- The ASP exposes a new API that streams round and out-of-round transactions as they're completed
+- The SDK provides a similar API, allowing clients to offer live updates on incoming and outgoing transactions
+- This feature enables more responsive and dynamic applications built on Ark
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:53]: This doesn't mean much

File: blog/2024-11-20-release-v0.4.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+This feature enhances the usability of Ark for users with intermittent connectivity, ensuring they don't miss out on the benefits of the network.
+
+### 🔄 No Required Settlement
+
+We've streamlined the transaction process:
+
+- VTXOs received out of round no longer need to be settled to be spent in another out-of-round transaction
+- This improvement reduces friction and increases the speed of off-chain transactions
+
+### 📡 Live Transaction Feed
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:47]: Better to say Streaming/Notificatiions?

File: blog/2024-11-20-release-v0.4.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+Real-time updates are now a reality with our new live transaction feed:
+
+- The ASP exposes a new API that streams round and out-of-round transactions as they're completed
+- The SDK provides a similar API, allowing clients to offer live updates on incoming and outgoing transactions
+- This feature enables more responsive and dynamic applications built on Ark
+
+This is a mock of what an ideal Ark Server would do, ie broadcast messages for round and out of round transactions via Nostr.
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:55]: What does this mean?

File: blog/2024-11-20-release-v0.4.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+## 🛠️ ASP Improvements
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:57]: Server

File: blog/2024-11-20-release-v0.4.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+- Connection of the internal Bitcoin wallet to bitcoind with ZMQ
+- Auto-unlock feature for the internal wallet
+- Increased testing surface for improved reliability
+- Consolidation of APIs, including renaming of RPCs and message fields for better consistency
+
+These improvements contribute to a more robust and reliable Ark Server.
+
+## 🚀 Getting Started
+
+To start leveraging these exciting new features in Ark v0.4, update your installations and check out our [updated documentation](https://arkdev.info) for detailed guides on how to implement and use these new capabilities.
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:71]: Link to the usage page for the server

File: blog/2024-11-20-release-v0.4.md (hunk 1 above)
Comment (altafan) [reply blog/2024-11-20-release-v0.4.md:3]: i didn't get this

File: blog/2024-11-20-release-v0.4.md (hunk 5 above)
Comment (altafan) [reply blog/2024-11-20-release-v0.4.md:55]: wanted to say that now tx streaming is done through a Server API, while the plan is to send these messages over nostr

File: blog/2024-11-20-release-v0.4.md (hunk 3 above)
Comment (altafan) [reply blog/2024-11-20-release-v0.4.md:53]: ok i drop the last line

File: blog/2024-11-20-release-v0.4.md (hunk 5 above)
Comment (tiero) [reply blog/2024-11-20-release-v0.4.md:55]: No remove it not needed

**Review by tiero — CHANGES_REQUESTED** [review]

File: blog/2024-11-20-release-v0.4.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+### 🔐 New Address Encoding
+
+We've made a significant change to our address encoding format. Instead of encoding user and Server public keys, we now encode the VTXO output script (Taproot key) and Server public key. This shift unlocks a world of possibilities:
+
+- The Server can create a VTXO for any kind of Ark address, including custom ones
+- Enable an ecosystem of services built on top of Ark
+- Increase flexibility for developers and users alike
+
+This change paves the way for more innovative applications and use cases within the Ark network. You can do your own experiments by interacting with our public Ark server [server_url] <!-- TODO: add url -->
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:28]: Do we want to do this?

Comment (tiero) [reply blog/2024-11-20-release-v0.4.md:28]: also I would say 

```suggestion
This change paves the way for more innovative applications and use cases built on Bitcoin with the Ark Protocol.
```

File: blog/2024-11-20-release-v0.4.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+## 🔎 What's Next
+
+As we continue to develop Ark, here are two key areas we're focusing on for future releases:
+
+### Full Implementation of Market Hours
+
+Our next major step is to complete the implementation of the Market Hours feature. A key component of this will be the introduction of round fees. Currently, the Ark server bears the full cost of round transactions. However, we're planning to shift this model so that round participants will share the cost of these transactions among themselves. 
+
+This change will make the advantages of Market Hours more apparent to users. By concentrating transactions during specific periods and sharing the costs, participants should be able to benefit from lower fees. This should provide a clear incentive for users to take advantage of Market Hours, ultimately leading to a more efficient and cost-effective network for everyone involved.
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:78]:
```suggestion
This change will make the advantages of Market Hours more apparent to users. By concentrating transactions during specific periods and sharing the costs, participants should be able to benefit from lower fees. This should provide a clear incentive for users to take advantage of Market Hours, ultimately leading to a more efficient and cost-effective Ark for everyone involved.
```

File: blog/2024-11-20-release-v0.4.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+
+### Enhanced VTXO Tree Signing Scheme
+
+We're also working on updating our VTXO tree signing scheme. This change is designed to [unlock liquidity before shared output expiration](https://arkdev.info/blog/unlock-liquidity-before-shared-output-expiration), which should improve the overall sustainability of the Ark network.
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:82]:
```suggestion
We're also working on updating our VTXO tree signing scheme. This change is designed to [unlock liquidity before shared output expiration](https://arkdev.info/blog/unlock-liquidity-before-shared-output-expiration), which should improve the overall sustainability of the Ark Server.
```

File: blog/2024-11-20-release-v0.4.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+description: Release ARK v0.4, featuring new address encoding, Ark notes, and intro to market hours.
+image: /img/ark-release-v0.4.jpg
+slug: ark-release-v0.4
+tags: [ark, clark, bitcoin, layer2, blockchain, nostr]
+title: "ARK v0.4 | New Address Encoding, Ark Notes, and announcing Market Hours"
+---
+
+<!-- TODO: add image -->
+
+**Greetings, Arkonauts! 🚀**
```
Comment (tiero) [inline blog/2024-11-20-release-v0.4.md:12]:
```suggestion
```

**Pull request merged**